src/app/pages/things/opcua/opcua.component.scss

```diff
@@ -5,3 +5,8 @@ button {
 nb-list-item {
   word-wrap: break-word;
 }
+
+.noPad {
+  padding-left: 1px;
+  padding-right: 1px;
+}
```

Comment:
`noPad` suggests that there is no padding. Plus, use the full word here, like `noPadding`, it's short and readable directly from the html template code.